sdk/react/widgets/AirPlayView.js

```diff
@@ -1,6 +1,14 @@
+import PropTypes from 'prop-types';
 import {
-  requireNativeComponent, 
+  requireNativeComponent,
 } from 'react-native';
 
+const iface = {
```

Comment:
it seems that `iface` is not used anywhere

Reply:
Thanks for you attention! Fixed